vscode-dotty/test/extension.test.ts

```diff
@@ -12,10 +12,10 @@ import * as vscode from 'vscode';
 import * as dotty from '../src/extension';
 
 // Defines a Mocha test suite to group tests of similar kind together
-suite("vscode-dotty tests", function () {
+//suite("vscode-dotty tests", function () {
```

Comment:
Shouldn't be necessary anymore.

Reply:
That was added by accident. I removed it now.